hadoop-cloud/pom.xml

```diff
@@ -201,7 +201,7 @@
      enables store-specific committers.
     -->
     <profile>
-      <id>hadoop-3.2</id>
```

Comment:
Sorry I'm not very familiar with the maven profile. Are you saying `-Phadoop-3.2` can also match this profile?